tgui/packages/tgui/interfaces/DnaConsole.js

```diff
@@ -1190,6 +1190,7 @@ const GeneticMakeupBuffers = (props, context) => {
     isViableSubject,
     makeupCapacity = 3,
     makeupStorage,
+    geneticMakeupCooldown,
```

Comment:
Keep the alphabetical order of the other properties.

Reply:
Assuming I know the alphabet without having to recite the ABC song.

Reply:
nobody does, but you can select all the lines and press f9 in vsc 🙂

Reply:
Can we make the ~~automated pedant~~ linter do or catch this?